Sources/NIO/System.swift

```diff
@@ -535,6 +556,50 @@ internal extension Posix {
     }
 }
 
+internal extension Posix {
+    static func cmsgFirstHeader(inside msghdr: UnsafePointer<msghdr>?) -> UnsafeMutablePointer<cmsghdr>? {
+        return sysCmsgFirstHdr(msghdr)
+    }
+    
+    static func cmsgNextHeader(inside msghdr: UnsafeMutablePointer<msghdr>?, from: UnsafeMutablePointer<cmsghdr>?)
```

Comment:
Let's break the line in the arguments as well.

Reply:
These pointers are forbidden to be `nil` (asserts in the C layer), so let's forbid `nil` here as well.

Reply:
Should we rename `from` to `after`?